<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>page_customer</class>
 <widget class="QWidget" name="page_customer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>950</width>
    <height>800</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>950</width>
    <height>800</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>用户界面</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_7">
   <item row="0" column="0" colspan="2">
    <widget class="QWidget" name="widget_8" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QWidget" name="widget_7" native="true">
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>16777215</height>
         </size>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="maximumSize">
            <size>
             <width>45</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>16</pointsize>
            </font>
           </property>
           <property name="text">
            <string>用户：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="UserName">
           <property name="maximumSize">
            <size>
             <width>150</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>16</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Customer</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <family>宋体</family>
          <pointsize>36</pointsize>
         </font>
        </property>
        <property name="text">
         <string>欢迎光临</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="minimumSize">
         <size>
          <width>68</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>68</width>
          <height>50</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">border-image: url(:/keli.jpg);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_2">
        <property name="maximumSize">
         <size>
          <width>90</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>退出登录</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_4">
        <property name="maximumSize">
         <size>
          <width>90</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>退出</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QListWidget" name="listWidget">
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>16777215</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="currentIndex">
      <number>2</number>
     </property>
     <widget class="QWidget" name="page">
      <layout class="QGridLayout" name="gridLayout">
       <item row="4" column="1">
        <widget class="QWidget" name="widget_3" native="true">
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QLabel" name="label_7">
            <property name="font">
             <font>
              <pointsize>14</pointsize>
             </font>
            </property>
            <property name="text">
             <string>购买商品数量</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="Input_sum">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>44</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>18</pointsize>
             </font>
            </property>
            <property name="minimum">
             <number>1</number>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QPushButton" name="btn_Buy">
         <property name="text">
          <string>购买</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QPushButton" name="btn_addcart">
         <property name="text">
          <string>加入购物车</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0" colspan="2">
        <widget class="QLabel" name="label_3">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>65</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>宋体</family>
           <pointsize>22</pointsize>
          </font>
         </property>
         <property name="text">
          <string>购物中心</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="7" column="1">
        <widget class="QPushButton" name="btn_clear">
         <property name="text">
          <string>重置</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QWidget" name="widget_2" native="true">
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="0" column="0">
           <widget class="QLabel" name="label_6">
            <property name="font">
             <font>
              <pointsize>14</pointsize>
             </font>
            </property>
            <property name="text">
             <string>购买商品名称</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLineEdit" name="Input_Name">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>44</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>18</pointsize>
             </font>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QWidget" name="widget" native="true">
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="0">
           <widget class="QLabel" name="label_5">
            <property name="font">
             <font>
              <pointsize>14</pointsize>
             </font>
            </property>
            <property name="text">
             <string>购买商品编号</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLineEdit" name="Input_Number">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>44</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>18</pointsize>
             </font>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="pushButton">
         <property name="minimumSize">
          <size>
           <width>192</width>
           <height>40</height>
          </size>
         </property>
         <property name="text">
          <string>查询今日商品</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" rowspan="7">
        <widget class="QTableView" name="tableView">
         <property name="minimumSize">
          <size>
           <width>550</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>550</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_2">
      <layout class="QGridLayout" name="gridLayout_6">
       <item row="0" column="0" colspan="2">
        <widget class="QLabel" name="label_8">
         <property name="font">
          <font>
           <family>宋体</family>
           <pointsize>22</pointsize>
          </font>
         </property>
         <property name="text">
          <string>我的购物车</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QTableView" name="cart_tableView">
         <property name="maximumSize">
          <size>
           <width>700</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QWidget" name="widget_4" native="true">
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0">
           <widget class="QPushButton" name="show_cart">
            <property name="minimumSize">
             <size>
              <width>40</width>
              <height>40</height>
             </size>
            </property>
            <property name="text">
             <string>查看我的购物车</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QPushButton" name="btn_buy">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>40</height>
             </size>
            </property>
            <property name="text">
             <string>单个购买</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QPushButton" name="btn_buyAll">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>40</height>
             </size>
            </property>
            <property name="text">
             <string>一键购买</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QPushButton" name="btn_delete">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>40</height>
             </size>
            </property>
            <property name="text">
             <string>删除商品</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_3">
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>10</y>
         <width>801</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>宋体</family>
         <pointsize>22</pointsize>
        </font>
       </property>
       <property name="text">
        <string>我的订单</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QTableView" name="record_tableview">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>571</width>
         <height>651</height>
        </rect>
       </property>
      </widget>
      <widget class="QWidget" name="widget_5" native="true">
       <property name="geometry">
        <rect>
         <x>580</x>
         <y>130</y>
         <width>251</width>
         <height>341</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="1" column="0">
         <widget class="QLineEdit" name="InputPdtName">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>40</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>20</pointsize>
           </font>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_10">
          <property name="font">
           <font>
            <family>宋体</family>
            <pointsize>22</pointsize>
           </font>
          </property>
          <property name="text">
           <string>需要查询的商品</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QWidget" name="widget_6" native="true">
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QPushButton" name="btn_search">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>25</height>
              </size>
             </property>
             <property name="text">
              <string>查询</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btn_cancleSearch">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>25</height>
              </size>
             </property>
             <property name="text">
              <string>取消</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QPushButton" name="btn_searchAll">
       <property name="geometry">
        <rect>
         <x>590</x>
         <y>60</y>
         <width>241</width>
         <height>40</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>40</height>
        </size>
       </property>
       <property name="text">
        <string>查询所有商品</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_deleteRecord">
       <property name="geometry">
        <rect>
         <x>590</x>
         <y>520</y>
         <width>241</width>
         <height>61</height>
        </rect>
       </property>
       <property name="text">
        <string>删除订单(退货)</string>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
